crawl-ref/source/player.cc

```diff
@@ -8845,6 +8846,8 @@ bool temperature_effect(int which)
         default:
             return false;
     }
+    else
+    return false;
```

Comment:
this return needs to be indented as well, otherwise it appears that it runs after the else, which it doesn't.